src/growth.jl

```diff
@@ -13,12 +13,16 @@ abstract type GrowthRule{R,W} <: CellRule{R,W} end
     ExponentialGrowth{R}(; rate, timestep)
     ExponentialGrowth{R,W}(; rate, timestep)
 
-Exponential growth based on a growth rate data, using exact solution.
+Exponential growth of population size N based on an intrinsic growth rate ``r``, using the exact solution between timesteps ``t`` and ``t-1``:
+
+```
+N_t = N_{t-1}e^{r t}
```

Comment:
You can use latex syntax for this:
https://juliadocs.github.io/Documenter.jl/stable/man/latex/#latex_syntax

I think it needs ```math